File: src/codes/Users_Manual/link/simulator_2.py
~from pyautd3 import Controller, AUTD3
~from pyautd3.link.simulator import Simulator
~autd: Controller[Simulator] = Controller.builder([AUTD3([0.0, 0.0, 0.0])]).open(Simulator.builder(8080))
autd.link.update_geometry(autd.geometry)

File: src/codes/Users_Manual/link/simulator_2.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Gain;
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Utils;
~var autd = Controller.Builder([new AUTD3(Vector3.Zero)]).Open(Simulator.Builder(8080));
autd.Link.UpdateGeometry(autd.Geometry);